### PR TITLE
[Rustbook] add next/prev section links in the book -- using js

### PR DESCRIPTION
**Pull request by jatinn**
Added javascript code to insert next/prev links in the rust book.
Related Issue - https://github.com/rust-lang/rust/issues/20835

### CODE REVIEW

**Comment by steveklabnik** [comment]
Thanks @jatinn !

@Gankro , what do you think of this PR?

File: src/rustbook/css.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
```
Comment (Gankro) [inline src/rustbook/css.rs]: Don't increment these, they're useless.

**Comment by Gankro** [comment]
Would you be able to build rustbook with these changes and host a copy for us to review the changes in production?

**Comment by jatinn** [comment]
Sure here you go http://jatinn.github.io/rust/README.html
Also should i go back and change the copyright year back to 2014? I thought in the contribution notes it mentioned updating it.

**Comment by Gankro** [comment]
@jatinn copyright stuff is in flux. See #21481 for discussion

**Comment by Gankro** [comment]
This doesn't work if I visit: http://jatinn.github.io/rust/

**Comment by Gankro** [comment]
Otherwise this looks great!

**Comment by Gankro** [comment]
Although I *would* appreciate some comments noting what this code is *for*, and maybe desciribing the high-level logic ("search the TOC for this URL, and then grab the predecessor/successor, and toss them at the bottom of the page")

**Comment by jatinn** [comment]
changes made

**Comment by Gankro** [comment]
@jatinn one last nit: we have a style guide for comments to be `//` style. Change that, squash the commits into one, and r=me

**Comment by jatinn** [comment]
r=@Gankro

**Comment by Gankro** [comment]
r+ e371d rollup

**Comment by Gankro** [comment]
Thanks a ton!

**Comment by Gankro** [comment]
fUCK

**Pull request merged**